docs/reference/juju-cli/list-of-juju-cli-commands/model-config.md

```diff
@@ -307,7 +307,7 @@ The following keys are available:
       description: |-
         SSH allowlist is a comma-separated list of CIDRs from
         which machines in this model will accept connections to the SSH service.
-        Currently only the aws, gce, openstack providers support ssh-allow
+        Currently only the aws & openstack providers support ssh-allow
```

Comment:
gce supports it too

Reply:
I didn't change the source here -- merely accepted the `.md` autogenerated from it when I built the docs. Looks like a forward merge gone wrong? Either way, I'll fix it now, thanks!